pkg/endpoint/policy.go

```diff
@@ -43,9 +43,9 @@ func (e *Endpoint) checkEgressAccess(owner Owner, opts option.OptionMap, dstID p
 
 	switch owner.GetPolicyTree().Allows(&ctx) {
 	case policy.ACCEPT, policy.ALWAYS_ACCEPT:
-		opts[opt] = true
+		opts[opt] = "enabled"
```

Comment:
So we can carry more than just boolean's. Things like the size of the connection tracking table, and so on. Undecided yet whether we should hardcode those things in the API itself as done for some other settings now or treat it as a generic key value map.